pom.xml

```diff
@@ -269,7 +269,19 @@
         <dependency>
             <groupId>uk.gov.pay</groupId>
             <artifactId>testing</artifactId>
-            <version>1.0.0-7</version>
+            <version>1.0.0-95d787ff5931ee0d5e54afd7fbc3834cf772dad4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.json</groupId>
+            <artifactId>javax.json-api</artifactId>
+            <version>1.1.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>1.1.4</version>
```

Comment:
This being required because `pay-java-commons` includes EclipseLink JPA 2.7.3 rather than full EclipseLink 2.6.4 and upgrading to EclipseLink 2.7.3 means we need this dependency now for Pact tests (see, for example, https://github.com/alphagov/pay-connector/pull/723 for the equivalent change in connector)